dali/test/python/test_operator_decoders_audio.py

```diff
@@ -50,16 +50,19 @@ def create_test_files():
 rate1 = 16000
 rate2 = 12999
 
+
 class DecoderPipeline(Pipeline):
   def __init__(self):
-    super(DecoderPipeline, self).__init__(batch_size=8, num_threads=3, device_id=0,
-                                          exec_async=True, exec_pipelined=True)
+    super().__init__(batch_size=8, num_threads=3, device_id=0, exec_async=True, exec_pipelined=True,
+                     output_dtype=[types.INT16, types.INT16, types.INT16, types.FLOAT,
+                                   types.FLOAT, types.FLOAT, types.FLOAT, types.FLOAT],
+                     output_ndim=[2, 2, 1, 1, 0, 0, 0, 0])
```

Comment:
Just a side-note, that this being separate arguments (and separate from output specification) isn't the nicest looking thing we did.